build.sh

```diff
@@ -314,6 +314,15 @@ if [ ${BUILD_BE} -eq 1 ] ; then
                     -Dabsl_DIR=${STARLET_INSTALL_DIR}/third_party/lib/cmake/absl \
                     -DgRPC_DIR=${STARLET_INSTALL_DIR}/third_party/lib/cmake/grpc \
                     -Dprometheus-cpp_DIR=${STARLET_INSTALL_DIR}/third_party/lib/cmake/prometheus-cpp \
+                    -DCURL_LIBRARY=${STARLET_INSTALL_DIR}/third_party/lib/libcurl.a \
```

Comment:
don't try to use starlet libcurl, starRocks has its `libcurl.a` built and imported in CMakeLists.txt

Reply:
changed to use libcurl of starrocks